src/login/logind-action.h

```diff
@@ -21,7 +21,7 @@ typedef enum HandleAction {
 
 typedef struct ActionTableItem ActionTableItem;
 
-#define handle_action_valid(x) (x && (x < _HANDLE_ACTION_MAX))
+#define handle_action_valid(x) (x >= 0 && (x < _HANDLE_ACTION_MAX))
```

Comment:
Please drop the nested parens in the right operand… It doesn't make sense to have them on just one operand.